dockers/docker-fpm/bgpd.conf.j2

```diff
@@ -14,6 +14,7 @@ log facility local4
 ! enable password {# {{ en_passwd }} TODO: param needed #}
 {% endblock system_init %}
 !
+{% if minigraph_bgp_asn != None %}
```

Comment:
Yes. You're right. But it's unusual to see something != None in python. Usually we write something is not None.
I don't know the right patter for Jinja2.
I found there're 'none' constant is Jinja2 templates http://jinja.pocoo.org/docs/2.9/templates/#expressions

Also I found a special function none() (see the same document).

So my concert here: I don't want to have an expression in our code which might be deprecated.